src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -51,137 +49,113 @@ public void validateSubscription(Subscription subscription) throws SubscriptionV
         if (subscription.getNotificationType().equals("REST_POST")) {
             this.RestPostMediaType(subscription.getRestPostBodyMediaType());
         }
-        // List<Requirement> reqList = subscription.getRequirements();
-        // for (int i=0; i < reqList.size(); i++) {
-        // this.validateJmespath(reqList.get(i).getConditions().get(0).getJmespath());
-        // }
-        LOG.info("Validating of subscription " + subscription.getSubscriptionName() + " finished successfully.");
+        LOGGER.debug("Validating of subscription " + subscription.getSubscriptionName() + " finished successfully.");
     }
```

Comment:
This code was out-commented code was kept due to we have, I think, a story for writing JmePath validator.
At the time when this validator class was implement we didn't know how to validate JmePath field. So we created a, I think, an own story for this.
If we implement this implementation later this code need to be rewritten, if its removed.
You can mail and ask about if we are going to implement a JmePath format validator.

Reply:
it just a loop which invoke method without realization line 182-186, which also was deleted. Todo is presented